src/services/ExperienceService.ts

```
@@ -0,0 +1,190 @@
+        throw error;
+      }
+    }
+  
+    // Keep the certificate methods for backwards compatibility
+    static async addCertificate(token: string, certificateData: any) {
+      try {
+        const formData = new FormData();
+        formData.append('title', certificateData.title);
+        formData.append('file', certificateData.file);
+  
+        const res = await fetch(`${this.BASE_URL}/experiences`, {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)